### DISCUSSION TITLE
Can we use any gremlin client with GraphScope?

### DISCUSSION THREAD

**Discussion by kesavkolla** [alibaba/GraphScope#2497] [Q&A]
I want to use my rust code talk to GraphScope. I want to use https://github.com/wolf4ood/gremlin-rs library to connect to GraphScope so that I can use graph traversal. Is it something workable?

**Comment by longbinlai** [comment]
Thank you for using GraphScope. I think it is totally okay as long as it is connected with official Gremlin server. We will have a look to confirm and come back to you later.
